qutebrowser/completion/completionwidget.py

```diff
@@ -216,15 +216,17 @@ def _next_page(self, upwards):
             else:
                 return self.model().first_item()
         else:
-            element_height = self.visualRect(idx).height()  # Finds Height of each CompletionView elements
+            # Finds Height of each CompletionView elements
+            element_height = self.visualRect(idx).height()  
             page_length = int(self.height()/element_height)
 
         while True:
             if upwards:
-                for x in range(page_length-1):  # Ensures bottom element stays at top of next page
+                # Ensures bottom element stays at top of next page
+                for _x in range(page_length-1):  
                     idx = self.indexAbove(idx)
             else:
-                for x in range(page_length-1):
+                for _x in range(page_length-1):
```

Comment:
just `for _ in range`. That seems to be the most established convention in this repo.